src/app/screens/BoardsList/Components/AddBoardModal/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import AddBoardModal from './layout';
+import './styles.css';
+
+class AddBoardModalContainer extends Component {
+  state = { visible: false };
+
+  openModal = () => {
+    this.setState({
+      visible: true
+    });
+  };
```

Comment:
I'd like to compact this in one line if possible. What do you think ? @sbalay @pabloferro @IgnacioAbadie

Reply:
I think it would also look nicer in one line but I'd keep it this way.
I'm ok with any syntax that the linter approves, mainly because it's an automated review. If we start asking people to change the code layout, we should be consistent and ask everyone the same in every pull request. That's a lot of effort from the reviewer taking a closer look to the code, and for the developer that needs to correct that comments which, at some point, are subjective.

Reply:
👍